tests/tests.yaml

```diff
@@ -13,4 +13,5 @@ packages:
   - amulet
```

Comment:
Do we actually need amulet and juju-deployer here any longer? @ktsakalozos I know you didn't change this but might know better than us about the requirements for packages here.